modules/redis/options.go

```diff
@@ -30,6 +32,8 @@ func (o Option) Customize(*testcontainers.GenericContainerRequest) error {
 
 // WithTLS sets the TLS configuration for the redis container, setting
 // the 6380/tcp port to listen on for TLS connections and using a secure URL (rediss://).
+// It also creates a temporary directory to store the TLS certificates, which is removed
+// when the container is terminated.
```

Comment:
suggestion: I would remove, its exposing internal workings the user shouldn't care about.